tests/Endpoints/InsightsTest.php

```diff
@@ -124,7 +122,7 @@ public function testGetInsight(): void
         $this->assertObjectHasAttribute('best_practices', $result->counts);
         $this->assertObjectHasAttribute('security', $result->counts);
         $this->assertObjectHasAttribute('performance', $result->counts);
-        foreach ($result->counts as $name => $response) {
+        foreach ((array)$result->counts as $name => $response) {
```

Comment:
This is necessary to pass phpstan tests, since `$result->counts` is an object and implicit object to array conversion is frowned upon.